plugin/lspconfig.lua

```diff
@@ -117,26 +117,25 @@ end, {
 
 api.nvim_create_user_command('LspStop', function(info)
   local current_buf = vim.api.nvim_get_current_buf()
-  local server_name, force
+  local server_id, force
   local arguments = vim.split(info.args, '%s')
   for _, v in pairs(arguments) do
     if v == '++force' then
       force = true
-    end
-    if v:find '%(' then
-      server_name = v
+    elseif v:find '^[0-9]$' then
```

Comment:
there can be simply `if tonumber(v)` ? if it no number it will return nil.

Reply:
Thank you, I'm not a `lua` programmer.

Reply:
Uhmm... `find` is faster than `tonumber`

```lua
local now = os.clock()
tonumber('10')
tonumber('10a')
print(os.clock() - now) -- 1.2e-05
local v = '10'
local v1 = '10a'
now = os.clock()
v:find '^[0-9]$'
v1:find '^[0-9]$'
print(os.clock() - now) -- 3e-06
```

I prefer `find`

Reply:
make sense looks like tonumber will do some memory alloc.